MAINTAINERS.md

```diff
@@ -0,0 +1,5 @@
+Maintainers of this repository and their focus areas:
+
+    Siddon Tang, tl@pingcap.com; @SiddonTang: Project Lead, Raft, Coprocessor, Transaction, Storage, Roadmap
```

Comment:
If you remove the `   ` (you start preformatted text here) at the start here (Maybe `>` instead?) it should link to their accounts for you.